desktop/apps/consignments2/components/describe_work/index.js

```diff
@@ -84,134 +98,169 @@ function DescribeWork (props) {
         </div>
-        <div className={b('row-item')}>
-          <Field name='year' component={renderTextInput}
-            item={'year'}
-            label={'Year*'}
-          />
+        <div className={b('row')}>
+          <div className={b('row-item')}>
+            <Field name='category' component={renderSelectInput}
+              item={'category'}
+              label={'Category*'}
+              options={categoryOptions}
+            />
+          </div>
+          <div className={b('row-item')}>
+            <Field name='year' component={renderTextInput}
+              item={'year'}
+              label={'Year*'}
+            />
+          </div>
         </div>
-      </div>
-      <div className={b('row')}>
-        <div className={b('row-item')}>
-          <Field name='height' component={renderTextInput}
-            item={'height'}
-            label={'Height*'}
-          />
-        </div>
-        <div className={b('row-item')}>
-          <Field name='width' component={renderTextInput}
-            item={'width'}
-            label={'Width*'}
-          />
+        <div className={b('row')}>
+          <div className={b('row-item')}>
+            <Field name='medium' component={renderTextInput}
+              item={'medium'}
+              label={'Medium*'}
+            />
+          </div>
         </div>
-        <div className={b('row-item')}>
-          <Field name='depth' component={renderTextInput}
-            item={'depth'}
-            label={'Depth'}
-          />
+        <div className={b('row')}>
+          <div className={b('row-item')}>
+            <Field name='height' component={renderTextInput}
+              item={'height'}
+              label={'Height*'}
+            />
+          </div>
+          <div className={b('row-item')}>
+            <Field name='width' component={renderTextInput}
+              item={'width'}
+              label={'Width*'}
+            />
+          </div>
+          <div className={b('row-item')}>
+            <Field name='depth' component={renderTextInput}
+              item={'depth'}
+              label={'Depth'}
+            />
+          </div>
+          <div className={b('row-item')}>
+            <Field name='dimensions_metric' component={renderSelectInput}
+              item={'dimensions_metric'}
+              label={'Units*'}
+              options={['in', 'cm']}
+            />
+          </div>
         </div>
-        <div className={b('row-item')}>
-          <Field name='dimensions_metric' component={renderSelectInput}
-            item={'dimensions_metric'}
-            label={'Units*'}
-            options={['in', 'cm']}
-          />
-        </div>
-      </div>
-      <div className={b('row', {'border-bottom': true})}>
-        <div className={b('row-item')}>
-          <Field name='edition' component={renderCheckboxInput}
-            item={'edition'}
-            label={'This is an edition'}
-          />
+        {
+          hasEditionValue && (
```

Comment:
Again, not actually that many changes here except this additional `hasEditionValue` check which conditionally shows/hides the "edition number" and "edition size" fields